docs/theory/statistics.rst

```diff
@@ -1,10 +1,174 @@
+network after a series of experiments.
+
+More specifically, the statistical model is a joint probability distribution
+:math:`\pi: \Theta \times Y\rightarrow [0,1]` that defines the probability
+density :math:`\pi(\theta, y)` of any possible combination of unknown
+parameters :math:`\theta` and observations :math:`y`. This model is written
+explicitly as a Stan program.
+
+Given a kinetic model and a set of observations :math:`y`, Maud uses Stan's
+adaptive Hamiltonian Monte Carlo algorithm to draw samples from the posterior
+distribution :math:`p(\theta\mid y)`. Each draw contains a configuration of
+unknown parameter values. Quantitative questions about the metabolic network
+can be answered by interrogating the ensemble of posterior draws.
+
+
+Structural Information
+=====================================================
+
+Maud assumes that the following structural information is known in advance:
+
+- The volume of each compartment and the metabolites it houses
+- The network stoichiometry, i.e. the proportions in which each reaction in the
+  network creates and destroys metabolites and which enzymes catalyse which
+  reactions. This information is encapsulated in a stoichiometric matrix
+  :math:`S` with a row for each metabolite-in-compartment and a column for each
+  enzyme.
+- Which metabolites-in-compartment modify which enzymes and how
+- Which metabolites-in-compartments are 'balanced', i.e. their concentrations
+  do not change when the system is in a steady state.
+
+We refer to this structural information collectively as a kinetic model. See
+[LINK] for a detailed description of these from a scientific point of view.
+
+The kinetic model defines a system of ODEs - one differential equation for each
+metabolite-in-compartment - with the rate of change of each
+metabolite-in-compartment :math:`m_i` described by the following equation:
+
+.. math::
+
+  \frac{dm_{i}}{dt} = \sum_{j} S_{ij} v_{j}(\theta, \mathbf{m})
+
+When the system is at a steady state, all the balanced
+metabolites-in-compartments have unchanging concentrations, so their entries in
+the equation above are zero.
+
+For a given set of parameters, enzyme concentrations and initial metabolite
+conditions, there should be a single steady state balanced metabolite
+concentration and set of fluxes.
+
+The kinetic model's role in Maud's statistical model is to connect latent
+parameters - i.e. :math:`\theta` above - with measureable quantities,
+i.e. :math:`\mathbf{m}` and :math:`\mathbf{v}`. 
+
+
+Experimental Information
+========================
+
+Maud represents information from experiments that measure enzyme concentrations
+and metabolite concentrations using the following regression model, where
+:math:`y` is the observation and :math:`\hat{y}` is the unobserved true value
+of the experimentally measured quantity:
+
+.. math::
+
+   y \sim LogNormal(\log(\hat{y}), \sigma)
+
+Measurements of reaction fluxes use the following similar regression model:
```

Comment:
I have a question about experimental measurements. Most people fit a summary statistic for the mean and standard deviation. By definition this assumes a normal distribution and we are assuming a log-normal distribution. It may be inconvenient but I would like to move to individual measurements rather than normal summaries which we then try and approximate with a log-normal distribution

Reply:
It's an interesting question how we translate an actual experiment result into something that can go into Maud. My general opinion is that we should make a method that takes in the most un-summarised possible data, as otherwise these normal/lognormal issues are likely to cause problems. I think it will probably be worth doing a bit of summarising as this should make it possible to save a few likelihood evaluations, but it needs to be done in a smart way and should definitely be (pre-processing stage) Maud's job rather than the user's.

Whatever method we choose, I think the measurement part of the Stan model will still look like this - a measured value `y` and a log-scale error standard deviation `\sigma`.

Reply:
I agree. I just think it should be an actual measured value. Is it compatible with multiple measurements?

Reply:
Yeah it should be - for multiple measurements of the same quantity I think we just need multiple experiments